map/src/main/kotlin/edu/artic/map/rendering/MapItemRenderers.kt

```diff
@@ -81,7 +88,12 @@ class SpacesMapItemRenderer(articMapAnnotationDao: ArticMapAnnotationDao)
     private val textMarkerGenerator: TextMarkerGenerator  by lazy { TextMarkerGenerator(context) }
 
     override fun getItems(floor: Int, displayMode: MapDisplayMode): Flowable<List<ArticMapAnnotation>> {
-        return articMapAnnotationDao.getTextAnnotationByTypeAndFloor(ArticMapTextType.SPACE, floor = floor.toString()) // TODO: switch to int
+        return when(displayMode) {
+            is MapDisplayMode.Search<*> ->
+                listOf<ArticMapAnnotation>().asFlowable()
```

Comment:
There seems to be a lot of these `listOf<T>().asFlowable()` in this form. Perhaps we could pull those out into a common function, or maybe even a constant object. @agrosner, what do you think?